numba_cuda/numba/cuda/codegen.py

```diff
@@ -200,11 +189,44 @@ def get_cubin(self, cc=None):
             linker.add_ptx(ptx.encode())
 
         for path in self._linking_files:
-            linker.add_file_guess_ext(path)
+            linker.add_file_guess_ext(path, ignore_nonlto)
```

Comment:
If user still installs pynvjitlink via `patch_numba_linker`, this call would throw because of mismatched number of argument.

Reply:
Synced offline: users should not use `patch_numba_linker` anymore.